dojo/tools/meterian/parser.py

```diff
@@ -0,0 +1,139 @@
+import json
+from urllib.parse import urlparse
+
+from dojo.models import Finding
+
+
+class MeterianParser(object):
+
+    def get_scan_types(self):
+        return ["Meterian Scan"]
+
+    def get_label_for_scan_types(self, scan_type):
+        return scan_type
+
+    def get_description_for_scan_types(self, scan_type):
+        return "Meterian JSON report output file can be imported."
+
+    def get_findings(self, report, test):
+        findings = []
+        report_json = self.parse_json(report)
+
+        security_reports = report_json["reports"]["security"]["reports"]
+        for single_security_report in security_reports:
+            findings += self.do_get_findings(single_security_report, test)
+
+        return findings
+
+    def do_get_findings(self, single_security_report, test):
+        findings = []
+        language = single_security_report["language"]
+        for dependency_report in single_security_report["reports"]:
+
+            lib_name = dependency_report["dependency"]["name"]
+            lib_ver = dependency_report["dependency"]["version"]
+            finding_title = lib_name + ":" + lib_ver
+            for advisory in dependency_report["advices"]:
+
+                severity = self.get_severity(advisory)
+                finding = Finding(
+                    title=finding_title,
```

Comment:
There is a date in the report.
Could you consider adding the `date` attribute to the findings?

Reply:
Sure, I will add that.